frontend/packages/kubevirt-plugin/src/components/modals/vm-description-modal/vm-description-modal.tsx

```diff
@@ -13,6 +13,7 @@ import { VMLikeEntityKind } from '../../../types';
 import { getDescription, getVMLikeModel } from '../../../selectors/selectors';
 import { getUpdateDescriptionPatches } from '../../../k8s/patches/vm/vm-patches';
 
+// TODO: should be moved under kubevirt-plugin/src/style.scss
```

Comment:
Looking for opinions: will we encapsulate styles with components or will we maintain single SASS hierarchy starting with `frontend/public/style.scss` (or `kubevirt-plugin/src/style.scss`).

The encapsulation has clearly its benefits, but we do not develop a library here. And the rest of the code follows the later approach.